components/EndowedPositions/EndowedPositionsPagination.tsx

```diff
@@ -14,6 +15,23 @@ export const EndowedPositionsPagination = ({
   totalPages,
   focusOnPageChangeId,
 }: EndowedPositionsPaginationProps) => {
+  const searchParams = useSearchParams();
+
+  // Smooth scroll when page parameter changes
+  useEffect(() => {
+    if (focusOnPageChangeId && currentPage > 1) {
+      const el = document.getElementById(focusOnPageChangeId);
+      if (el) {
+        el.scrollIntoView({
+          behavior: 'smooth',
+          block: 'start',
+          inline: 'nearest',
+        });
+        el.focus();
+      }
+    }
+  }, [searchParams, focusOnPageChangeId, currentPage]);
```

Comment:
Nice. Just a couple nitpicks:

- Can we toss in our reduced motion check?
- Let's prevent the `focus()` call from triggering a competing scroll jump

```suggestion
  // Smooth scroll when page parameter changes
  useEffect(() => {
    if (focusOnPageChangeId && currentPage > 1) {
      const el = document.getElementById(focusOnPageChangeId);
      if (el) {
        const reduceMotion = !!window.matchMedia('(prefers-reduced-motion: reduce)')?.matches;
        el.scrollIntoView({
          behavior: reduceMotion ? 'instant' : 'smooth'
          block: 'start',
          inline: 'nearest',
        });
        el.focus({ preventScroll: true });
      }
    }
  }, [searchParams, focusOnPageChangeId, currentPage]);
```

Reply:
Oh yay thank you.